client/src/components/posts/PostDetails.js

```diff
@@ -72,6 +76,28 @@ export const PostDetails = ({ loggedInUser }) => {
             <p>
                 Published: {post.publishDateTime}, Author: {post.userProfile.fullName}
             </p>
+            <div className="postReactions">
+                {reactions.map((r) => (
```

Comment:
This is good, given what the API method currently exposes. It seems like the next step would be, instead of getting postreactions for a post, to get all reactions, with postreactions for a specific post, and do the count that way, so that we include all reactions (even if the count will be 0).

Reply:
As it stands, the DTO's data isn't helping you avoid much logic in the front end.

Reply:
Yeah, I see that, I just feel like the DTO isn't doing enough work. I'll resolve this, I just wanted to point out that we haven't really solved the problem.